library/core/src/char/methods.rs

```diff
@@ -390,8 +390,9 @@ impl char {
     /// assert_eq!('❤'.escape_unicode().to_string(), "\\u{2764}");
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
+    #[rustc_const_stable(feature = "const_char_escape_unicode", since = "1.50.0")]
     #[inline]
-    pub fn escape_unicode(self) -> EscapeUnicode {
+    pub const fn escape_unicode(self) -> EscapeUnicode {
```

Comment:
When would this ever be called in a const context given that it returns an iterator?

Reply:
Good point, I didn't really consider the practicality for making it const. I'm not sure about the downsides of making it callable in const context, but I'm totally fine with removing it. It'll probably be a good while before const iterators happen anyway.

Reply:
> I'm not sure about the downsides of making it callable in const context, but I'm totally fine with removing it.

Yeah, please remove the iterator functions.